translator-src/nu/validator/htmlparser/cpptranslate/CppVisitor.java

```diff
@@ -1280,6 +1290,15 @@ public void visit(MethodCallExpr n, LocalSymbolTable arg) {
         } else if ("transition".equals(n.getName())
                 && n.getScope() == null) {
             visitTransition(n, arg);
+        } else if ("checkChar".equals(n.getName())
+                && n.getScope() == null) {
```

Comment:
Does null scope mean static method?

Reply:
I don't actually recall with 100% certainty, since I copypasted from existing code, but I believe it means that the call isn't qualified by an expression followed by a dot.

Thanks for the review.